tiup: add 3 docs disable/display/list (#5176)
pingcap/docs: tiup/tiup-component-cluster-display.md

---
title: tiup cluster display
---

# tiup cluster display

> If you want to see the operation status of each component in the cluster, it is obviously inefficient to log in to each machine one by one. Therefore, tiup-cluster provides the `tiup cluster display` command to efficiently complete this task.

Reviewer: 登录到每台机器应该是翻译成 "login to each machine" 还是 "log in to each machine" 呀？

Reviewer 2: `log in` 吧，动词短语，`login` 是名词

## Syntax

```shell
tiup cluster display <cluster-name> [flags]
```

> `<cluster-name>`: the name of the cluster to operate. If you forget the cluster name, you can check it by the [cluster list](/tiup/tiup-component-cluster-list.md).

Reviewer 2: ```suggestion
`<cluster-name>`: the name of the cluster to operate on. If you forget the cluster name, you can check it in the [cluster list](/tiup/tiup-component-cluster-list.md).
```

> `<cluster-name>`: the name of the cluster to operate on. If you forget the cluster name, you can check it in the [cluster list](/tiup/tiup-component-cluster-list.md).

Reviewer: ```suggestion
`<cluster-name>`: the name of the cluster to operate on. If you forget the cluster name, you can check it with the command [cluster list](/tiup/tiup-component-cluster-list.md).
```

## Options

### --dashboard

> - By default, all node information of the entire cluster is displayed. After adding the `--dashboard` option, only dashboard information is displayed.

Reviewer: ```suggestion
- By default, all node information of the entire cluster is displayed. With the `--dashboard` option, only dashboard information is displayed.
```

- Data type: `BOOLEAN`
- Default: false

### -N, --node

- Data type: `strings`
- Default: `[]`, which means all nodes.

> - Specifies the node to query. If not specified, it means all nodes. The value of this option is a comma-separated list of node IDs. The node ID is the first column of the [cluster status](/tiup/tiup-component-cluster-display.md) table.

Reviewer 2: ```suggestion
- Specifies the node to query. If not specified, all nodes are queried. The value of this option is a comma-separated list of node IDs. The node ID is the first column of the [cluster status](/tiup/tiup-component-cluster-display.md) table.
```

> **Note:**
>

> > If `-R, --role` are specified at the same time, then the service status in their intersection is queried.

Reviewer 2: same as previous comments

### -R, --role strings

- Data type: `strings`
- Default: `[]`, which means all roles.

> - Specifies the role to query. If not specified, it means all roles. The value of this option is a comma-separated list of node roles. The role is the second column of the [cluster status](/tiup/tiup-component-cluster-display.md) table.

Reviewer 2: ```suggestion
- Specifies the role to query. If not specified, all roles are queried. The value of this option is a comma-separated list of node roles. The role is the second column of the [cluster status](/tiup/tiup-component-cluster-display.md) table.
```

> **Note:**
>

> > If `-N, --node` are specified at the same time, then the service status in their intersection is queried.

Reviewer 2: ditto
